plugins/modules/ec2_vpc_nat_gateway_info.py

```diff
@@ -122,7 +196,7 @@ def get_nat_gateways(client, module, nat_gateway_id=None):
 def main():
     argument_spec = dict(
         filters=dict(default={}, type='dict'),
-        nat_gateway_ids=dict(default=[], type='list', elements='str'),
+        nat_gateway_ids=dict(default=[], type='list', elements='str')
```

Comment:
Please keep the trailing comma here.  By having a comma already on the last item in this dict, the next time someone adds an option to the module they won't have to add it to this line which reduces the number of lines in the diff when someone reviews it.

Reply:
@jillr Thank you for reviewing. I added the trailing comma.